lib/MusicBrainz/Server/Data/ReleaseGroup.pm

```diff
@@ -252,6 +252,33 @@ sub find_by_track_artist
     );
 }
 
+sub find_by_artist_credit
+{
+    my ($self, $artist_credit_id, $limit, $offset) = @_;
+
+    my $query = "SELECT DISTINCT " . $self->_columns . ",
+                    rgm.first_release_date_year,
+                    rgm.first_release_date_month,
+                    rgm.first_release_date_day,
+                    rg.name COLLATE musicbrainz AS name_collate,
+                    array(
+                      SELECT name FROM release_group_secondary_type rgst
+                      JOIN release_group_secondary_type_join rgstj
+                        ON rgstj.secondary_type = rgst.id
+                      WHERE rgstj.release_group = rg.id
+                      ORDER BY name ASC
+                    ) secondary_types
+                 FROM " . $self->_table . "
+                 WHERE rg.artist_credit = ?
+                 ORDER BY
+                   rg.type, secondary_types,
+                   rgm.first_release_date_year,
+                   rgm.first_release_date_month,
+                   rgm.first_release_date_day,
+                   rg.name COLLATE musicbrainz";
```

Comment:
Same as above...I don't think it makes sense to sort by secondary information you can't see. The queries should be faster that way too.